server/render/index.js

```diff
@@ -89,6 +90,10 @@ export function serverRender( req, res ) {
 		context.i18nLocaleScript = '//widgets.wp.com/languages/calypso/' + context.lang + '.js';
 	}
 
+	if ( moment.locale() !== 'en' ) {
```

Comment:
For my own understanding, if you could clarify those, it'll be great:

1. Where do we change the output of `moment.locale()` ?
2. Can it be anything other than english on the server?

Thank you!

Reply:
1.`moment.locale()` is initialized by `i18n-calypso` by means of the router setup: https://github.com/Automattic/wp-calypso/blob/master/client/login/index.web.js#L27 when calling `/log-in` with a locale different than `en`

2. Yes, is set by `i18n-calypso`: https://github.com/Automattic/i18n-calypso/blob/master/lib/index.js#L193

Reply:
@yurynix I applied your suggestions: removed `moment` import and used `context.lang` to initialize moment locale file name.

It seems to work correctly :+1: